internal/support/log.go

```diff
@@ -0,0 +1,60 @@
+package support
+
+import (
+	"context"
+	"fmt"
+	"io"
+	"log/slog"
+	"os"
+)
+
+func NewDefaultLogger(logLevel slog.Level) *slog.Logger {
+	return slog.New(NewDefaultTextHandler(
+		os.Stdout,
+		logLevel,
+	))
+}
+
+type DefaultTextHandler struct {
+	w     io.Writer
+	level slog.Level
+}
+
+func NewDefaultTextHandler(w io.Writer, level slog.Level) *DefaultTextHandler {
+	return &DefaultTextHandler{w: w, level: level}
+}
+
+func (h *DefaultTextHandler) Enabled(_ context.Context, level slog.Level) bool {
+	return level >= h.level
+}
+
+func (h *DefaultTextHandler) Handle(_ context.Context, r slog.Record) error {
+	// Format time as YYYY-MM-DD hh:mm:ss
+	timeStr := r.Time.Format("2006-01-02 15:04:05")
+
+	// Map slog.Level to uppercase string (WARNING for WARN, etc.)
+	levelStr := r.Level.String()
+
+	// Start building the log line
+	line := fmt.Sprintf("%s %s %s", timeStr, levelStr, r.Message)
+
+	// Append all key-value attributes
+	r.Attrs(func(attr slog.Attr) bool {
+		line += fmt.Sprintf(" %s=%v", attr.Key, attr.Value.Any())
+		return true
+	})
+
+	// Write the line
+	_, err := fmt.Fprintln(h.w, line)
+	return err
+}
+
+func (h *DefaultTextHandler) WithAttrs(_ []slog.Attr) slog.Handler {
```

Comment:
Having this empty means that attributes pre-configured in the `With` function (in `baseLogger` where we do `base.loggerInstance.With("operation_id", operationID)` are not going to be printed.

I'm not sure how easy is to fix this. If it is complex. I would be fine even removing the `With` usage in the baseOperator.

PD: Testing this is easy. Just run workbench and you will see that `operation_id` is not printed